exercises/accumulate/AccumulateTests.elm

```diff
@@ -12,18 +15,21 @@ square x =
 
 tests : Test
 tests =
-    suite "Accumulate"
+    describe "Accumulate"
         [ test "[]] Accumulate"
-            (assertEqual [] (accumulate square []))
+            (\() -> Expect.equal [] (accumulate square []))
```

Comment:
Yeah... To be honest I was able to do 99% of this via regex flavored find/replace because I was keeping the existing closing paren at the end of each test. Switching to that style will require a bit more work. That's really my strongest objection :)